File: src/main/java/um/haberes/rest/model/dto/FormularioAsignacionCargo.java
/**
 * 
 */
package um.haberes.rest.model.dto;

import java.io.Serializable;
import java.util.List;

import um.haberes.rest.model.Categoria;
import um.haberes.rest.model.Dependencia;
import um.haberes.rest.model.Facultad;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class FormularioAsignacionCargo implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -1349554321974212244L;

	private List<Categoria> categorias;
	private List<Categoria> categoriasAsignables;
	private List<Dependencia> dependencias;
	private List<Facultad> facultades;

}

File: src/main/java/um/haberes/rest/kotlin/model/dto/FormularioAsignacionCargo.kt
package um.haberes.rest.kotlin.model.dto

import um.haberes.rest.kotlin.model.Categoria
import um.haberes.rest.kotlin.model.Dependencia
import um.haberes.rest.kotlin.model.Facultad

data class FormularioAsignacionCargo(

    var categorias: List<Categoria?>? = null,
    var categoriasAsignables: List<Categoria>? = null,
    var dependencias: List<Dependencia>? = null,
    var facultades: List<Facultad>? = null

)
